doc/wintabs.txt

```diff
@@ -200,6 +200,32 @@ default: ' '
                 '[]' and ')(' are some of the good ones. It's recommended that
                 they have the same length as separators.
 
+*g:wintabs_ui_show_vimtab_name*
+values: 0, 1 or 2
+default: 0
+                This controls how Vim tabs will be displayed.
+                0: show a number of the tab.
+                1: show a name of the tab.
+                2: show number and name of the tab. (<number>:<tab_name>).
```

Comment:
I'm against this option (number+name) as this intentionally isn't an option for the buffer list. The reason being that once the tab is named, its tab number becomes quite useless (looping through tabs or mouse click is faster than using tab number).